lib/metadata/linux/LinuxUtils.rb

```diff
@@ -115,5 +115,29 @@ def self.parse_chkconfig_list(lines)
 
       return ret
     end
+
+    def self.parse_systemctl_list(lines)
+      ret = []
+      return ret if lines.nil? || lines.empty?
```

Comment:
Make the changes below, and you can delete `ret` and change the above line to:

`return [] if lines.nil? || lines.empty?`